modules/templates/helper.go

```diff
@@ -81,6 +81,9 @@ func NewFuncMap() []template.FuncMap {
 		"LoadTimes": func(startTime time.Time) string {
 			return fmt.Sprint(time.Since(startTime).Nanoseconds()/1e6) + "ms"
 		},
+		"AllowedReactions": func() []string {
```

Comment:
Hmm... Maybe if we're going to do this we should just allow templates to query setting?

Reply:
would be a solution ... but then we can refactor a lot :)

Reply:
I also agree that we could just name function "Settings" and use it here and later in other places where needed